dll/win32/advpack/files.c

```diff
@@ -454,7 +458,7 @@ HRESULT WINAPI DelNodeW(LPCWSTR pszFileOrDirName, DWORD dwFlags)
     HRESULT ret = E_FAIL;
     
     TRACE("(%s, %d)\n", debugstr_w(pszFileOrDirName), dwFlags);
-
+    
```

Comment:
please revert

Reply:
> revert

What is the purpose of this diff?

Reply:
Removing extra trailing whitespace that has been added into this PR for unknown reasons

Reply:
You got it wrong: my PR restores these whitespaces which have been removed for no reason.